cmd/sealer/cmd/image/push.go

```diff
@@ -34,18 +38,29 @@ var exampleForPushCmd = `
 func NewPushCmd() *cobra.Command {
 	pushCmd := &cobra.Command{
 		Use:   "push",
-		Short: "push ClusterImage to remote registry",
+		Short: "push sealer image to remote registry",
 		// TODO: add long description.
 		Long:    longNewPushCmdDescription,
 		Example: exampleForPushCmd,
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			engine, err := imageengine.NewImageEngine(options.EngineGlobalConfigurations{})
+
+			login, err := checkIsLogin(pushOpts.Authfile)
 			if err != nil {
 				return err
 			}
-			pushOpts.Image = args[0]
-			return engine.Push(pushOpts)
+
+			if login {
```

Comment:
i dont agree add check whether user is login in here . what if user push to registry that is not require authentication.